.agents/skills/use-cursor/SKILL.md

```
@@ -0,0 +1,111 @@
+## high vs xhigh
+
+- rank tasks on complexity scale: 1-10
+- high: 1-4
+- xhigh: 4-6
+- greater than 6: Never use grok
+
+## Prototype
+
+`$CA create-chat` → `$SID`
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)